share/models/meta.py

```diff
@@ -1,17 +1,41 @@
+import furl
+import json
+
 from django.db import models
 from django.db import IntegrityError
 from django.contrib.postgres.fields import JSONField
 
 from share.models.base import ShareObject
 from share.models.fields import ShareForeignKey, ShareURLField, ShareManyToManyField
-from share.apps import ShareConfig as share_config
 
 
-__all__ = ('Venue', 'Award', 'Tag', 'Link', 'Subject')
+__all__ = ('Venue', 'Award', 'Tag', 'Subject', 'Identifier', 'Relation')
 
 # TODO Rename this file
 
 
+class Identifier(ShareObject):
+    # https://twitter.com/berniethoughts/
+    url = ShareURLField(unique=True)
+
+    # https://twitter.com/
+    def get_base_url(self):
+        return furl(self.url).origin
+
+
+class Relation(ShareObject):
+    with open('./share/models/relation-types.json') as fobj:
+        # TODO add label to file
+        RELATION_TYPES = [(t['key'], t['uri']) for t in json.load(fobj)]
```

Comment:
I think it is important to store the uri to allow for additional ontologies to be used in the future.  I do have one question here in thinking how you have a particular string key per item...

For example, say 3-6 months from now we want to support an additional ontology of relationships (especially if it is being loaded from another system), and there are overlaps in relationships between the two different ones (e.g., they both have a 'cites' relationship).  I assume with a typical dictionary/hash you would only have one value per key.  So, I am wondering if the key should be changed to include something like '_spar'.   Another option might be a key pointing to a list of possible uri's (of course more complicated and maybe not something to do now).

Are you storing the key as value in the relationship table or is a link to a foreign that points to the item in the relationship table?  If it is a foreign key, than that may be good enough as you could then reference the key or value at any point.  

Thoughts?